auth-web/src/components/auth/staff/continuation-in/StaffContinuationInTable.vue

```diff
@@ -0,0 +1,310 @@
+<template>
+  <div>
+    <v-form class="fas-search item-active-search">
+      <v-row
+        dense
+        class="row-margin"
+      >
+        <v-col
+          sm="12"
+          cols="6"
+        >
+          <transition name="slide-fade">
+            <v-data-table
+              :headers="headerAccounts"
+              :items="businesses"
+              :footer-props="{
+                itemsPerPageOptions: paginationOptions
+              }"
+              hide-default-header
+              fixed-header
+              :loading="isTableLoading"
+              :mobile-breakpoint="0"
+            >
+              <!-- Loading -->
+              <template #loading>
+                <div
+                  class="py-8 loading-datatable"
+                >
+                  Loading items...
+                </div>
+              </template>
+
+              <!-- No data -->
+              <template #no-data>
+                <div class="py-8 no-data">
+                  No data available
+                </div>
+              </template>
+
+              <!-- Headers (two rows) -->
+              <template #header="{}">
+                <thead class="v-data-table-header">
+                  <!-- First row has titles. -->
+                  <tr class="header-row-1">
+                    <th
+                      v-for="(header, i) in headerAccounts"
+                      :key="i"
+                      class="font-weight-bold"
+                    >
+                      {{ header.text }}
+                    </th>
+                  </tr>
+
+                  <!-- Second row has search boxes. Search and filter to be implemented -->
+                  <tr class="header-row-2 mt-2 px-2">
```

Comment:
The search and filter is not implemented yet

Reply:
Right. Will be done in bcgov/entity#21983.